### PR TITLE
@eessex => Mobile auth pages QA

### PR DESCRIPTION
**Pull request by l2succes**
Addresses:
 
- [[Mobile Auth QA] Change redirect link for back button for '/login' and '/signup' static route](https://artsyproduct.atlassian.net/browse/GROW-786)
- [Mobile modals - Redirect legacy mobile auth urls to new auth static pages](https://artsyproduct.atlassian.net/browse/GROW-496)

Tasks
- [x] Make the update from /sign_up -> /signup and /log_in -> /login
- [x] Ensure that when updating, we're using proper parameters (ie destination instead of redirectTo for post-onboarding)

Depends on: https://github.com/artsy/reaction/pull/1006

### CODE REVIEW

**Comment by l2succes** [comment]
Need update reaction with a version that includes this PR https://github.com/artsy/reaction/pull/1006 first

File: src/desktop/apps/auth2/components/MobileAuthStatic.tsx [hunk 1]
```diff
@@ -38,15 +36,8 @@ export class MobileAuthStatic extends React.Component<Props> {
...
-              if (typeof window !== 'undefined') {
-                window.location.href = submitUrls.facebook + `?${authQueryData}`
-              }
-            }}
-            onTwitterLogin={() => {
-              if (typeof window !== 'undefined') {
-                window.location.href = submitUrls.twitter + `?${authQueryData}`
-              }
+            onBackButtonClicked={() => {
+              window.location.href = options.redirectTo || '/'
```
Comment (eessex) [inline src/desktop/apps/auth2/components/MobileAuthStatic.tsx]: I'd imagine we still want to check if the window exists here for ssr

File: src/mobile/apps/auth/routes.coffee [hunk 2]
```diff
@@ -23,14 +23,19 @@ module.exports.login = (req, res) ->
   locals =
     redirectTo: redirectUrl(req)
     action: req.query.action
-
-  if locals.action
+  
+  if sd.NEW_AUTH_MODAL
+    res.redirect "/login?#{qs.stringify(locals)}"
```
Comment (eessex) [inline src/mobile/apps/auth/routes.coffee]: We should be testing these routes.

**Comment by l2succes** [comment]
@eessex I addressed your feedback, let me know if there's anything else.

**Comment by eessex** [comment]
Looks good, merging

**Pull request merged**